lib/src/kbaserelationengine/tools/RESKETools.java

```diff
@@ -99,17 +100,20 @@ public int execute() {
             printError("For config file " + a.configPath, e, a.verbose);
             return 1;
         }
-        return runEventGenerator(cfg, out, a.ref, a.verbose);
+        final List<Integer> wsBlackList = a.wsBlacklist == null || a.wsBlacklist.isEmpty() ?
```

Comment:
For higher convenience we can add support for workspace names in black list as well (maybe later).